static/js/components/dashboard/CourseDescription_test.js

```diff
@@ -31,80 +35,102 @@ export function findCourse(courseSelector: (course: Course, program: Program) =>
 }
 
 describe('CourseDescription', () => {
-  const now = moment();
+  let createCourseRun = (status: string, courseEndDate: string, courseStartDate: string) => {
```

Comment:
nitpick: it makes more sense to position the `courseStartDate` before the `courseEndDate` argument